lib/src/core/models/kanji.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:isar/isar.dart';
+
+part 'kanji.g.dart';
+
+@collection
+@Name("Kanjis")
+@JsonSerializable()
+class Kanji {
+  final int id;
+
+  final String kanji;
+  final int? nbrTraits;
+  final int? grade;
+  @JsonKey(name: "niveau_jlpt")
+  final int niveauJLPT;
+  @Default([])
+  final List<String> traductions;
+  @Default([])
+  @JsonKey(name: "lectures_on")
+  final List<String> lecturesOn;
+  @Default([])
+  @JsonKey(name: "lectures_kun")
+  final List<String> lecturesKun;
```

Comment:
Pour confirmer, traduction[0] est lu lectures_on[0] ou lectures_kon[0]?
Est-ce qu'on a une méthodes pour déterminer la bonne pronunciation par rapport a la traduction? 🤔

Reply:
Non, il n'y a rien pour faire le lien.
Il aurait fallu le savoir à la saisie des données dans la bd.

Reply:
Oui, je pense même pas que dans Kanji to kana, il y a un lien qui est indiqué 🤔 pour le moment on va laisser comme ca ecoute!

Reply:
En fait il y a des listes pour dire "ces lectures ON et/ou lectures KUN signifient cette liste de choses"